src/kata4.humanCatDogYears.js

```diff
@@ -1,3 +1,13 @@
-const humanCatDogYears = number => {};
-
+const humanCatDogYears = number => {
+    if (number === 1){
+        return [1,15,15];
+    }
+    if (number === 2) {
+        return [2, 24, 24];
+    } else {
+        const catYears = 24 + (number-2)*4;
+        const dogYears = 24 + (number-2)*5;
```

Comment:
good use of brackets here to separate the mathematical equations. I know mathematically there is an order in which operations are carried out, for example multiplication happens before addition if you have a statement like 5 x 4 + 3. Here is the order precedence for JS. Should help you to know when parenthesis are necessary or not :) https://www.dummies.com/article/technology/programming-web-design/general-programming-web-design/javascript-operator-precedence-254119